### PR TITLE
Support self-signed installation CAs

### PR DESCRIPTION
**Pull request by QuentinBisson**
Towards https://github.com/giantswarm/roadmap/issues/2956

### CODE REVIEW

**Comment by QuentinBisson** [comment]
Tested and is working fine on Goat

File: pkg/resource/grafana-agent-config/grafana-agent-config.go [hunk 1]
```diff
@@ -81,6 +81,9 @@ loki.write "default" {
 		username = nonsensitive(remote.kubernetes.secret.credentials.data["logging-username"])
 		password = remote.kubernetes.secret.credentials.data["logging-password"]
 	}
+	tls_config {
+		insecure_skip_verify = ` + fmt.Sprintf("%v", lc.IsInsecureCA()) + `
```
Comment (TheoBrigitte) [inline pkg/resource/grafana-agent-config/grafana-agent-config.go:85]: I assume we expect `true` or `false` here, using `%t` is preferable then.
```suggestion
		insecure_skip_verify = ` + fmt.Sprintf("%t", lc.IsInsecureCA()) + `
```

**Review by TheoBrigitte — COMMENTED** [review]
Looks good, just 1 remark

File: pkg/resource/grafana-agent-config/grafana-agent-config.go (hunk 1 above)
Comment (QuentinBisson) [reply pkg/resource/grafana-agent-config/grafana-agent-config.go:85]: Thanks :)

**Comment by marieroque** [comment]
Let me finish to validate on Vintage MC and WC before merge

**Comment by QuentinBisson** [comment]
Yours? Sure

**Comment by marieroque** [comment]
✅ Test ok on `gaia`

**Review by marieroque — APPROVED** [review]

**Pull request merged**